package.json

```diff
@@ -19,9 +20,13 @@
     "jsonwebtoken": "^8.5.1",
     "less": "^3.12.2",
     "lessc": "^1.0.2",
+    "md5": "^2.3.0",
+    "moment": "^2.29.0",
     "mongoose": "^5.9.15",
     "react": "^16.13.1",
+    "react-cookie": "^4.0.3",
     "react-dom": "^16.13.1",
+    "react-moment": "^0.9.7",
```

Comment:
I believe this hasn't been used anywhere?
The only place that used moment used the package above this?